ci-operator/step-registry/sdn/conf/multi-manifest/sdn-conf-multi-manifest-commands.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+cat > "${SHARED_DIR}/manifest_cluster-network-03-config.yml" << EOF
+apiVersion: config.openshift.io/v1
+kind: Network
+metadata:
+  name: cluster
+spec:
+  serviceNetwork:
+  - "172.30.0.0/16"
+  clusterNetwork:
+  - cidr: "10.128.0.0/14"
+    hostPrefix: 23
+  networkType: OpenShiftSDN
+  openshiftSDNConfig:
+    mode: Multitenant
+    vxlanPort: 4789
+    mtu: 1450
```

Comment:
@pecameron I don't htink we want to set mtu 1450; GCP should use a much bigger MTU. We can leave this out I think.

Reply:
Changed.